terraform/infra/ecr.tf

```diff
@@ -2,6 +2,7 @@ locals {
   repository_names = [
     "bags_api",
     "bag_indexer",
+    "bag_register",
```

Comment:
errr, how did we miss this?

Reply:
I think what happened is that we rejigged the ECR repo definitions at some point, and when we were rearranging the Terraform state the bag register got missed.